.ci/travis/travis.sh

```diff
@@ -342,6 +342,41 @@ no-exception-test-alot-of-project1)
       --config checks-nonjavadoc-error.xml --checkstyleVersion $CS_POM_VERSION
   ;;
 
+check-missing-files)
+  fail=0
+  list=($(cat pom.xml | sed '3 s/xmlns=".*"//g' | \
```

Comment:
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/pom-version-bump.sh#L24 it is better without sed

Reply:
Done. Thanks for the link.